lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -97,6 +97,7 @@ sub test_network_interface {
         }
         script_retry("ssh root\@$guest ifup $nic", delay => 30, retry => 3, timeout => 90);
     }
+    script_retry("ssh root\@$guest ip -o -4 addr list $nic | awk \"{print \\\$4}\" | cut -d/ -f1", delay => 90, retry => 9, timeout => 180);
```

Comment:
What is this about? Also be careful that you can get more than one IP as the output.

Reply:
Good question,

Reply:
> so, before get $addr under test_network_interface func, I try to get more than one IP as the output via script_retry here.

I don't see any multiple addresses check. This `script_retry` will finish even with one outputted address.

Reply:
After attached interface, guest system get the related IP address in time, there will be not any multiple addresses check here. But, we have big test matrix here. We can not let each guest system get IP address in time on each version sles vm hosts. our virtul network test have a big problem - depend on test environment (sles version - sles11, sles12+ and sles15 +, sles vm - kvm | xen, sles guest - pv, fv , osd lab network ... and so on)

Reply:
1) You are not checking for multiple addresses.
 2) The `script_retry` makes no sense here - `assert_script_run` should be enough.

Reply:
sure, change to use `assert_script_run`